.github/dependabot.yml

```diff
@@ -5,9 +5,10 @@ version: 2
 updates:
   # Maintain dependencies for npm ecosystem
   - package-ecosystem: 'npm'
-    # Checks all directories from the current layer and below recursively for package.json files
+    # Checks root and top level package directories.
     directories:
-      - '**/*'
+      - '/'
+      - '/packages/*'
```

Comment:
see   https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/controlling-dependencies-updated#defining-multiple-locations-for-manifest-files for reference